isabl_cli/app.py

```diff
@@ -936,6 +936,10 @@ def run_analyses(self, analyses, commit, force, restart, local):
                 ):
                     skipped_tuples.append((i, i["status"]))
                     continue
+                
+                elif restart and i.ran_by != system_settings.api_username:
+                    skipped_tuples.append((i, i["status"]))   
```

Comment:
What if instead of SKIPPED we make it INVALID showing a proper message saying something like: `Can't --restart as it was started by different user. Consider --force`?

Reply:
sounds great! Thanks for the suggestion, I'll make the changes

Reply:
@Keimya those are very good approaches. Both might work. I like approach 1. Let's try that. It make sense to add `invalid_run_tuples` to the run_analyses()` method. 

Thank you for looking deep into it. This is a very good addition